packages/itwinui-css/src/bridge.scss

```diff
@@ -311,26 +430,50 @@
 
     // StrataKit tab has no background or border
     border: none;
+    background-color: transparent;
   }
 
-  // StrataKit tabs content has no background or border
   .iui-tabs-content {
+    // StrataKit tabs content has no background or border
     background-color: transparent;
     border: none;
+    // StrataKit border color here is not equal to --iui-color-border's global mapping
+    border-color: var(--ids-color-border-neutral-muted);
+  }
+
+  /* ---------------------------------------------------------------------------- */
+  /* Tag */
+
+  .iui-tag {
+    // StrataKit tag background color is not iTwinUI's --iui-color-background global mapping
+    background-color: var(--ids-color-bg-neutral-base);
+  }
+
+  /* ---------------------------------------------------------------------------- */
+  /* Tile */
+
+  // To make buttons match Tile background color
+  .iui-tile-buttons button {
+    background-color: var(--iui-color-background);
   }
 
   /* ---------------------------------------------------------------------------- */
   /* ToggleSwitch */
 
   .iui-toggle-switch {
+    &:not(:checked) {
+      // StrataKit unchecked toggle switch background color is not iTwinUI's --iui-color-background global mapping
+      background-color: var(--ids-color-bg-page-base);
```

Comment:
This does not seem necessary. It matches the global mapping.

https://github.com/iTwin/iTwinUI/blob/8b6898b8e35942421437360c62f755f2f4d3a250/packages/itwinui-css/src/bridge.scss#L7

Can you go through all your changes and make sure there are no inaccuracies (perhaps leftovers from a pervious version of the PR)?